scrapers/EPS-epsom-and-ewell/polling_stations.py

```diff
@@ -0,0 +1,25 @@
+from lgsf.polling_stations.scrapers.arcgis_scraper import ArcGisScraper
+from lgsf.polling_stations.scrapers.xml_scrapers import GmlScraper
+
+
+class Scraper(GmlScraper):
```

Comment:
FWIW, I think this is really nice and readable. Not convinced that having two classes (`PollingScraper` and `DistrictsScraper`) would be any easier for a developer to work on. There might be other reasons to have them of course.